### PR TITLE
Fixing incorrect error message on Verify

### PR DESCRIPTION
**Pull request by ChaosInTheCRD**
The changed error message seems to have been copied from `cmd/sign.go` and is a confusing error to present to the user. Instead it should be specified that the policy file can't be opened.

### CODE REVIEW

**Review by jkjell — DISMISSED** [review]

**Comment by ChaosInTheCRD** [comment]
just fyi I am currently working on improving how we handle required flags. Maybe we should save the merge until I finish? shouldn't take long.

File: options/verify.go [hunk 1]
```diff
@@ -26,6 +28,10 @@ type VerifyOptions struct {
 	CAPaths              []string
 }
 
+var RequiredVerifyFlags = []string{
+	"policy",
```
Comment (jkjell) [inline options/verify.go:32]: I think the public key will also be required.

Comment (ChaosInTheCRD) [reply options/verify.go:32]: So for verify *either* `--publickey` or `--policy-ca` can be specified: https://github.com/in-toto/witness/blob/main/cmd/verify.go#L59-L61

Comment (adityasaky) [reply options/verify.go:32]: https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsOneRequired?

Comment (ChaosInTheCRD) [reply options/verify.go:32]: Ah - thank you for this @adityasaky I missed it!

**Comment by mikhailswift** [comment]
These changes look good. Can we get the merge conflicts sorted, change Aditya recommended, and lining fixed up so this can get merged?

File: options/run.go [hunk 2]
```diff
@@ -36,17 +36,31 @@ type RunOptions struct {
 	AttestorOptSetters       map[string][]func(attestation.Attestor) (attestation.Attestor, error)
 }
 
+var RequiredRunFlags = []string{
+	"step",
+	// we're going to require an out file path specified as stdout is an unreliable default
+	"outfile",
```
Comment (jkjell) [inline options/run.go:42]: If you're using archivista you don't need an outfile.

**Review by jkjell — APPROVED** [review]

**Pull request merged**